gradle.properties

```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.2.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 17
+cordaApiRevision = 17-julia2
```

Comment:
You'll need to rebase and bump the version number.
The suffix will need to be removed as well before merging.